docs/en/latest/plugins/prometheus.md

```diff
@@ -151,27 +153,17 @@ Or you can goto [Grafana official](https://grafana.com/grafana/dashboards/11719)
 * `etcd reachability`: A gauge type with a value of 0 or 1, representing if etcd can be reached by a APISIX or not, where `1` is available, and `0` is unavailable.
 * `Connections`: Various Nginx connection metrics like active, reading, writing, and number of accepted connections.
 * `Batch process entries`: A gauge type, when we use plugins and the plugin used batch process to send data, such as: sys logger, http logger, sls logger, tcp logger, udp logger and zipkin, then the entries which hasn't been sent in batch process will be counted in the metrics.
-* `Latency`: The per service histogram of request time and the overhead added by APISIX (request time - upstream response time).
+* `Latency`: The per service histogram of request time in different dimensions.
 
     Attributes：
 
     | Name      | Description |
     | ----------| ------------- |
-    | type      | Its value is fixed as `request`, which means HTTP request. |
+    | type      | The optional values are `apisix`, `upstream` and `request`, which means http latency caused by apisix, upstream, and in total. |
```

Comment:
"The value can be `apisix`, `upstream` or `request`, which means http latency caused by apisix, upstream, or their sum." would be better?

Reply:
Thanks for your suggestions and it is indeed more fluent! Fixed